modules/computer/boots.c

```diff
@@ -30,11 +30,11 @@ scan_boots_real(void)
     scan_os(FALSE);
 
     if (!computer->os->boots)
-      computer->os->boots = g_strdup_printf("[%s]\n", _("Boots"));
+      computer->os->boots = g_strdup_printf("");
     else
       return;
 
-    last = popen("last", "r");
+    last = popen("last 2>&1", "r");
```

Comment:
I'd rather just use `g_spawn_sync_with_pipes()` everywhere instead of `popen()`, which spawns a shell. The glib function will `fork()` and `exec()` by itself.